src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine+TransferInput.kt

```diff
@@ -44,11 +44,6 @@ fun TradingStateMachine.transfer(
     environment: V4Environment,
 ): StateResponse {
     if (staticTyping) {
-        if (type == null) {
-            val error = ParsingError.cannotModify("type")
-            return StateResponse(state, null, iListOf(error))
-        }
```

Comment:
are we removing because because type can be `null`?

Reply:
Yeah.. iOS is passing null there